src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import duke.task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+public class Storage {
+    String filePath;
+    PrintWriter writer;
+    TaskList ls;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void rewriteFile(TaskList ls) {
+        this.ls = ls;
+        try {
+            FileWriter fw = new FileWriter(filePath, false);
+            writer = new PrintWriter(fw);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Possible to combine into a single catch statement here
```suggestion
        } catch (FileNotFoundException | IOException e) {
            e.printStackTrace();
        } 
```